tools/clang/test/HLSLFileCheck/hlsl/control_flow/return/multi_ret.hlsl

```diff
@@ -0,0 +1,140 @@
+// RUN: %dxc -E main -fcgl -structurize-returns -T ps_6_0 %s | FileCheck %s
+
+// CHECK:entry
```

Comment:
Multiple dependencies on CHK build, like this one.  This is why it fails AppVeyor.